client/src/views/page-components/ProductDetailPage/ProductDetail/Carousel/CarouselCount/CarouselCount.js

```diff
@@ -0,0 +1,26 @@
+export default function CarouselCount({ $target, initialState }) {
+  /**
+   * {
+   *  total : 7,
+   *  current : 1,
+   * }
+   */
+  this.state = initialState;
+
+  this.$carouselCount = document.createElement('div');
+  this.$carouselCount.className = 'carousel__counter__container';
+  $target.appendChild(this.$carouselCount);
+
+  this.setState = (nextState) => {
+    this.state = nextState;
+    this.render();
+  };
+
+  this.render = () => {
+    this.$carouselCount.innerHTML = `<span class="carousel__counter">${this.state.current + 1} / ${
+      this.state.total
+    }</span>`;
+  };
+
```

Comment:
내부 로직을 함수로 분리했으면 어땠을까요 :)

```suggestion
  this.render = () => {
    const { current, total } = this.state;
    this.$carouselCount.innerHTML = `
        <span class="carousel__counter">
            ${getCarouselIndex(current, total)}
        </span>`;
  };

  const getCarouselIndex = (idx, total) => {
    return `${idx + 1} / ${total}`
  }
```

Reply:
더 알아보기가 쉽네요!! 감사합니다 :)